src/DiagramEngine.ts

```diff
@@ -29,6 +29,7 @@ export class DiagramEngine extends BaseEntity<DiagramEngineListener> {
 	canvas: Element;
 	paintableWidgets: {};
 	linksThatHaveInitiallyRendered: {};
+	maxNumberPointsPerLink: number;
```

Comment:
Most of the changes in this file are auto tslinting.

Real changes are: this new property added (along with a getter and setter further down below).